plugins/inventory/proxmox.py

```diff
@@ -360,7 +360,17 @@ def _get_vm_config(self, properties, node, vmid, vmtype, name):
                         agent_iface_key = self.to_safe('%s%s' % (key, "_interfaces"))
                         properties[agent_iface_key] = agent_iface_value
 
-                if config not in plaintext_configs and not isinstance(value, int) and all("=" in v for v in value.split(",")):
+                if config == "lxc":
+                    out_val = {}
+                    for key_list in value:
+                        k, v = key_list
+                        if k.startswith('lxc.'):
+                            k = k.replace('lxc.', '')
+                        out_val[k] = v
+                    value = out_val
+
+                if config not in plaintext_configs and not isinstance(value, int) and \
+                        not isinstance(value, dict) and all("=" in v for v in value.split(",")):
```

Comment:
Right now this is a new feature PR that includes a bugfix PR. For this to be a proper bugfix PR, you should only do something like:
```suggestion
                if config not in plaintext_configs and isinstance(value, string_types) and all("=" in v for v in value.split(",")):
```
(with `from ansible.module_utils.six import string_types`)

Reply:
Integrated.